lib/gui/components/safe-webview.js

```diff
@@ -196,16 +196,28 @@ class SafeWebview extends react.PureComponent {
   /**
    * @summary Forward console messages from the webview to analytics module
    * @param {Event} event - event object
+   *
+   * @example
+   * console.log({
+   *   type: 0,
+   *   message: 'Hello, World!'
+   * });
    */
   static consoleMessage(event) {
+    const LOG_LEVEL = 0;
+    const WARN_LEVEL = 1;
     const ERROR_LEVEL = 2;
 
-    if (event.level < ERROR_LEVEL) {
-      analytics.logEvent(event.message);
+    _.attempt(() => {
+      const data = JSON.parse(event.message);
 
-    } else if (event.level === ERROR_LEVEL) {
-      analytics.logException(event.message);
-    }
+      if (data.type === LOG_LEVEL || data.type === WARN_LEVEL) {
+        analytics.logEvent(data.message);
+
+      } else if (data.type === ERROR_LEVEL) {
+        analytics.logException(data.message);
+      }
+    });
```

Comment:
So, just to check I'm understanding this diff correctly:
`consoleMessage` takes an `event` object, which is expected to have a (possibly JSON-encoded) `.message` attribute? (and also previously a `.level` attribute, which is now ignored)
And then if the JSON message is decoded correctly (i.e. into `data`), it's then expected to have both `.type` and `.message` attributes?

Which I guess means the expected usage is something like:
```
consoleMessage({level:0, message: "{type: 0, message: 'Hello, World!'}"});
```